src/components/student/StudentsList.vue

```diff
@@ -32,6 +33,9 @@ function handleRemove(studentId) {
               {{ st.githubUsername }}
             </a>
           </td>
+          <td>
+            <a v-bind:href="'./students/' + st.id">Details</a>
```

Comment:
Since it's an internal link, here we could use a router link

```suggestion
            <router-link :to="{ name: 'student', params: { id: st.id} }">Details</router-link>
```

Reply:
Cool, didn't know that!